pkgs/by-name/ma/maa-x/package.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, stdenv
+, fetchzip
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+, android-tools
+, electron
+}:
+
+stdenv.mkDerivation rec {
+  pname = "maa-x";
+  version = "2.0.0-beta.14";
+
+  src = fetchzip {
+    url = "https://github.com/MaaAssistantArknights/MaaX/releases/download/v${version}/maa-x-linux-x64-${version}.zip";
+    hash = "sha256-FU/1CPxgi8GK1oZna+LZMc3J04Wn7Fph0c7CIqx1tIE=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  buildInputs = [
+    android-tools
+    electron
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/${pname}
+    cp -r -v resources/app/* $out/share/${pname}/
+    makeWrapper ${electron}/bin/electron $out/bin/MaaX \
+      --argv0 "MaaX" \
+      --add-flags "$out/share/${pname}/dist/main/index.cjs" \
+      --set LD_LIBRARY_PATH "$LD_LIBRARY_PATH:${stdenv.cc.cc.lib}/lib/"
+
+    sed -i '1a mkdir -p ~/.config/Electron\
+    rm -rfv ~/.config/Electron/platform-tools\
+    ln -s ${android-tools}/bin ~/.config/Electron/platform-tools' $out/bin/MaaX
```

Comment:
No, please don't.

Reply:
> No, please don't.

Has Asked upstream fixed it.

Reply:
maa-x has been abandoned by the upstream, so it has been deleted.